### PR TITLE
Thomas/place receiver cmake

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
After spending 30min trying to compile with scons, I decided to switch to make.
I don't know anything about cmake, but I could adapt a CMakeLists.txt from https://github.com/SeisSol/PUMGen/blob/master/CMakeLists.txt

### CODE REVIEW

File: place_receivers/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.9)
+
+project(place_receivers LANGUAGES C CXX)
+
+set(LOG_LEVEL "info" CACHE STRING "Log level for the code")
+set_property(CACHE LOG_LEVEL PROPERTY STRINGS "debug" "info" "warning" "error")
+
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
```
Comment (sebwolf-de) [inline place_receivers/CMakeLists.txt]: `cmake` directory is not checked in, but will be needed for `FindXYZ`.

**Review by sebwolf-de — CHANGES_REQUESTED** [review]
It speaks for itself that writing a new cmake script, is easier than fixing scons :D

**Comment by krenzland** [comment]
Looks good to me, can be merged once cmake director is included. Thanks!

> I don't know anything about cmake
You used to know nothing about cmake, now you're an expert as well ;)

File: place_receivers/CMakeLists.txt (hunk 1 above)
Comment (Thomas-Ulrich) [reply place_receivers/CMakeLists.txt]: ok there is no FindXYZ needed for this project.

**Review by sebwolf-de — APPROVED** [review]

**Pull request merged**